src/SupervisionSuccess/utils/deriveModelParamsFromCsvString.js

```diff
@@ -30,10 +32,29 @@ async function deriveModelParamsFromCsvString(string) {
         ...row
       }
     ) => {
+      const requiredFields = {
```

Comment:
Make sure to update this to include the `Data Year` column, after my latest comment on #23.